src/components/Reports/StaffExpenseReport/CategoryBreakdownDialog/CategoryBreakdownDialog.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableFooter,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { CancelButton } from 'src/components/common/Modal/ActionButtons/ActionButtons';
+import { useLocale } from 'src/hooks/useLocale';
+import { currencyFormat, dateFormat } from 'src/lib/intlFormat';
+import { Transaction } from '../StaffExpenseReport';
+
+export interface CategoryBreakdownDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  categoryName: string;
+  transactions: Transaction[];
+  totalAmount: number;
+}
+
+export const CategoryBreakdownDialog: React.FC<
+  CategoryBreakdownDialogProps
+> = ({ isOpen, onClose, categoryName, transactions, totalAmount }) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const theme = useTheme();
+
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={onClose}
+      maxWidth="md"
+      fullWidth
+      PaperProps={{
+        sx: {
+          maxHeight: '700px',
+        },
+      }}
+    >
+      <DialogTitle>
+        {categoryName} {t('Breakdown')}
+      </DialogTitle>
+      <DialogContent
+        sx={{
+          padding: theme.spacing(4),
+          overflow: 'hidden',
+          display: 'flex',
+        }}
+      >
+        <TableContainer>
+          <Table stickyHeader>
+            <TableHead>
+              <TableRow
+                sx={{
+                  backgroundColor: 'background.paper',
+                  fontWeight: 'bold',
+                }}
+              >
+                <TableCell>{t('Date')}</TableCell>
+                <TableCell>{t('Description')}</TableCell>
+                <TableCell align="right">{t('Amount')}</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {transactions
+                .sort((a, b) =>
```

Comment:
`.sort` modifies the original array. We will also want to memoize the sorted transactions to avoid having to resort on every render.

```suggestion
                .toSorted((a, b) =>
```